docs/source/guides/metrics.rst

```diff
@@ -0,0 +1,22 @@
+=======
+Metrics
+=======
+
+Metrics are quantitative assessments for measuring and conducting assessments of your system's performance.
+Metrics are vital in providing reliable and accurate diagnoses to your BentoService in real-time scenarios.
+
+BentoML enables users to define custom metrics with `Prometheus <https://prometheus.io/>`_ to bootstrap and fast-track one's monitoring journey.
+ 
+This article will dive into how to add custom metrics to monitor your BentoService and a quick tutorial on how one can incorporate custom metrics to a :ref:`concepts/runner:Custom Runner`.
+
+Tracking model latency with Prometheus and :ref:`concepts/runner:Custom Runner`
+-------------------------------------------------------------------------------
+
+We will build a custom histogram to track the latency of our custom Flax runner in this
```

Comment:
I guess this is incomplete?

Reply:
Yes this is wip, I haven't push it up yet